ngraph/core/src/op/shape_of.cpp

```diff
@@ -193,6 +193,12 @@ shared_ptr<Node> op::v0::ShapeOf::clone_with_new_inputs(const OutputVector& new_
 {
     check_new_args_count(this, new_args);
     auto new_shape_of = make_shared<op::v0::ShapeOf>(new_args.at(0));
+    NGRAPH_CHECK(new_shape_of.get(),
```

Comment:
What was the error here exactly? I'm just curious why only this op and TensorIterator have this extra check in their clone methods now.

Reply:
KW find issues that potentially `new_shape_of` can be nullptr.